src/status_im/ui/screens/profile/db.cljs

```diff
@@ -6,6 +6,8 @@
             [status-im.constants :as constants]
             [status-im.utils.homoglyph :as homoglyph]))
 
+(def account-profile-keys [:name :photo-path :status])
```

Comment:
Is this always used as a cursor? If so I think we should indicate it as such, e.g. `photo-status-path` or something

Reply:
it's not a cursor it is a list of keys to select from the profile when extracting profile